src/travis2docker/travis2docker.py

```diff
@@ -131,6 +130,9 @@ def _compute_env(data, _):
                 continue
             env_globals += " " + env_global
         env_globals = env_globals.strip()
+        psql_version = self.yml.get('addons', {}).get('postgresql')
```

Comment:
Use `get('addons') or {}` in order to consider the case where `get('addons')` returns `None`
I mean,

```python
my_dicc = {'addons': None}
my_dicc.get('addons', {}).get('postgresql')  # AttributeError: 'NoneType' object has no attribute 'get'
```

vs

```python
my_dicc = {'addons': None}
(my_dicc.get('addons') or {}).get('postgresql')
```

Reply:
Done. I didn't know yaml parsing could produce None values, but:
```
Addons:
```
will